### DISCUSSION TITLE
Taskbar Icon Spacing

### DISCUSSION THREAD

**Discussion by genericadm** [ramensoftware/windhawk-mods#154] [General]
How about a Mod to change the Taskbar icon spacing? I'm happily using "Taskbar height and icon size", but there's no setting to change the spacing. I'm talking about the pinned items to the taskbar (immediately right of the Windows logo icon), not the open apps icons.

**Comment by m417z** [comment]
Right now, there are the following solutions you can try:

*   Install the <a href="https://windhawk.net/mods/taskbar-labels" rel="nofollow ugc">Taskbar Labels for Windows 11</a> mod, and set a narrow width in settings. It's not 100% supported and tested, there might be issues, for example, when there are too many items on the taskbar. But generally I believe that it should work. Let me know if you encounter any issues with it.
*   Try <a href="https://github.com/ramensoftware/windhawk-mods/pull/145" rel="nofollow ugc">this updated version of the mod</a>. Let me know if you need help installing it.
